src/outputs/outputs_main.py

```diff
@@ -100,6 +100,10 @@ def run_outputs(
         )
         OutputMainLogger.info("Finished long form output.")
 
+    # Filter out records that answer "no R&D" for all subsequent outputs
+    tau_outputs_df = tau_outputs_df.copy().loc[~(tau_outputs_df["604"] == "No")]
+    outputs_df = outputs_df.copy().loc[~(outputs_df["604"] == "No")]
+
```

Comment:
@JenCheshire  I just updated this from ~(df[   which needed correcting, can you just check I've done the right thing? Then I can approve.

Reply:
@AnneONS Yes, that's right - thank you for spotting that!